### DISCUSSION TITLE
Unable to verify student status

### DISCUSSION THREAD

**Discussion by 60iTomi** [community/community#77827] [GitHub Education]
### Hi everyone,

I have been trying to get verified to Github Global Campus these past few days.
When trying to upload photo proof, I only get the option to take a picture of the document I'd like to submit. Fair enough, but when I click on the 'Take a picture' button, only the loading sign appears. This seems to persist even if I wait for it to load - though my browser does ask for camera permissions. I have tried this on two different desktops and my phone, both on- and off-campus.
Thanks for all answers in advance.

Best regards,
Tamas

**Comment by dakshithadissanayaka** [comment]
It appears that you're encountering an issue while trying to get verified on GitHub Global Campus, specifically when attempting to upload a photo as proof. If you're facing difficulties with the camera feature, here are some steps you can take to troubleshoot the issue:

Browser Compatibility:

Ensure that you are using a supported and up-to-date browser. GitHub may work best with the latest versions of browsers like Chrome, Firefox, Safari, or Edge.
Camera Permissions:

Verify that your browser has the necessary permissions to access your camera. Check your browser settings to ensure that it is allowed to use the camera.
Try a Different Browser:

Attempt to use a different web browser to see if the issue persists. Sometimes, browser-specific issues can be resolved by switching to an alternative browser.
Incognito/Private Mode:

Open an incognito or private browsing window and try uploading the photo from there. This helps rule out any potential conflicts with browser extensions or cached data.
Clear Browser Cache:

Clear your browser cache and cookies. Cached data might sometimes interfere with the proper functioning of certain features.
Browser Extensions:

Disable any browser extensions that might be affecting the camera functionality. Some extensions, especially those related to privacy or security, may interfere with certain website features.
Mobile Device:

If possible, try using a mobile device with a supported browser to upload the photo. Sometimes, mobile browsers provide a different experience.
